apps/frontend/src/libs/components/header/header.tsx

```diff
@@ -5,20 +5,22 @@ import { UserDto } from "~/libs/types/types.js";
 import styles from "./styles.module.css";
 
 const Header: React.FC = () => {
-	const user: UserDto = useAppSelector(
-		({ auth }) => auth.user?.user as UserDto,
+	const user: undefined | UserDto = useAppSelector(
```

Comment:
The header will be displayed only when the user is logged in, so we can use type assertion